pkg/vtpm/swtpm-vtpm/src/main.go

```diff
@@ -130,19 +202,23 @@ func main() {
 		os.Exit(1)
 	}
 
-	uds, err := net.Listen("unix", types.VtpmdCtrlSocket)
+	serviceLoop()
+}
+
+func serviceLoop() {
+	uds, err := net.Listen("unix", vtpmdCtrlSocketPath)
 	if err != nil {
 		log.Errorf("failed to create vtpm control socket: %v", err)
 		return
 	}
 	defer uds.Close()
 
-	// make sure we remove the socket file on exit
+	// Make sure we remove the socket file on exit.
 	sigs := make(chan os.Signal, 1)
 	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
 	go func() {
 		<-sigs
-		os.Remove(types.VtpmdCtrlSocket)
+		os.Remove(vtpmdCtrlSocketPath)
```

Comment:
Don't we have to clean some extra files here? Like the SWTPM socket, if it exists?